src/popup/router/components/UrlBadge.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <Badge :class="type ? `badge-${type}` : ''">
+    <template v-if="type === 'verified'">
+      <TickIcon />
+      {{ $t(`badges.${type}`) }}
+    </template>
+    <template v-if="type === 'untrusted'"> ! {{ $t(`badges.${type}`) }} </template>
```

Comment:
```suggestion
    <TickIcon v-if="type === 'verified'" />
    <template v-else-if="type === 'untrusted'"> ! </template>
    {{ $t(`badges.${type}`) }}
```

Reply:
> $t(`badges.${type}`)

Computed translation keys make more difficult to track unused/missed keys I would propose to avoid them.